service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CustomAsyncExceptionHandler.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.fpl.utils;
```

Comment:
Can we move that into `exceptions/handlers` package? That is definitely not a utility class.